### ISSUE TITLE
Using GitHub discussions

### ISSUE THREAD

**Issue by LinusU** [expressjs/discussions#128]
A lot of the issues we get on Multer is really people that want's help with doing specific setups, or integrating with another framework. The issue tracker isn't really the right place to discuss that, but maybe we could turn on GitHub Discussions and use that as a knowledge base for how to integrate/architect/etc?

I was recommended this by my friend @albinekb, who recommended looking at these two great examples:
- https://github.com/tannerlinsley/react-table/discussions
- https://github.com/zeit/next.js/discussions

**Comment by dougwilson** [comment]
I think that is pretty neat, but AFAICT there isn't a way to enable it in our repos at this time while it is in beta with only selected groups by GitHub.

**Comment by LinusU** [comment]
Ahh, hehe, seems like you are right 👍 

> Available in beta for public repositories soon

**Comment by gireeshpunathil** [comment]
I like the idea of actions that generally improves usage understanding as well as describes how a piece of code works. Given the technical problem with the above approach, how about improving the documentation of multer, that cover the specific aspects that you mentioned ? such as integration, architecture etc.? In addition, I would also like to see if we have a section that illustrates a complete life cycle events involved in a file upload operation - that might reduce a good amount of user issues in the repo IMO!

**Comment by wesleytodd** [comment]
I was told that I can get it enabled on ours if we want.  That said, if you take a look at them [in the npm RFCs repo](https://github.com/npm/rfcs/discussions), I am not sure it is great yet.  They converted many issues to discussions, and some are hard to follow now, also, I am not sure how the engagement has looked, we could ask @darcyclarke?

**Comment by sheepsteak** [comment]
GitHub Discussions is available on all public repositories now.

**Comment by bjohansebas** [comment]
We are already using GitHub Discussions :), well, in most of the repositories, where it has been needed.
